docs/index.rst

```diff
@@ -3,10 +3,10 @@ MicroPython documentation and references
 
 .. toctree::
 
-    wipy/quickref.rst
-    wipy/general.rst
-    wipy/tutorial/index.rst
     library/index.rst
     reference/index.rst
     genrst/index.rst
     license.rst
+    pyboard/quickref.rst
```

Comment:
If this is what's shown in the left navbar, then IMHO it's worth to have this under separate heading, like "Quick references".

Reply:
But it's not really quick access if it's further hidden under a sub menu.  And I tried to make the left navigation similar to the top index, which has the list like this in this order.

Reply:
Well, that simply doesn't scale to large number of "quick ref" docs. Perhaps it can be done later, but then again it's either biases against adding a quick ref for more ports, or will likely stay like that (long flat list) longer than necessary, leading to mess in the ToC.